commerce_coordinator/apps/core/signal_helpers.py

```diff
@@ -41,7 +41,7 @@ def format_signal_results(results):
     # The results of a send_robust are a tuple of a reference to the method called and the exception, if one was raised
     data = {}
     for receiver, response in results:
-        receiver_name = str(receiver)
+        receiver_name = receiver.__name__
```

Comment:
the old implementation was returning something like this.
```
{
    "<function order_created_save at 0x40091f2940>": {
        "response": "",
        "error": false
    }
}
```

here is an example response after the update.
```
{
    "order_created_save": {
        "response": "",
        "error": false
    }
}
```